_rules/aria-attr-defined-5f99a7.md

```diff
@@ -93,17 +93,15 @@ This `input` element with a role of `spinbutton` has multiple `aria-*` attribute
 
 #### Failed Example 1
 
-This `li` element with a role of `menuitemcheckbox` has an `aria-*` attribute which is not defined in [WAI-ARIA Specifications][].
+This `div` element with a role of `checkbox` has an `aria-not-checked` attribute which is not defined in [WAI-ARIA Specifications][].
 
 ```html
-<ul>
-	<li role="menuitemcheckbox" aria-not-checked="true">List Item</li>
-</ul>
+<div role="checkbox" aria-not-checked="true">All conditions are met</div>
 ```
 
 #### Failed Example 2
 
-This `div` element with a role of `searchbox` has an `aria-*` attribute (`aria-labelled`) which is not defined in [WAI-ARIA Specifications][].
+This `div` element with a role of `searchbox` has an `aria-` attribute (`aria-labelled`) which is not defined in [WAI-ARIA Specifications][].
```

Comment:
```suggestion
This `div` element with a role of `searchbox` has an `aria-labelled` attribute (instead of `aria-labelledby`) which is not defined in [WAI-ARIA Specifications][].
```